### PR TITLE
long chat desktop notifications

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

**Comment by michielbdejong** [comment]
You can try it out on https://chat.inrupt.app/

**Comment by Vinnl** [comment]
When I click the login button at the bottom:

image

I also don't see how I can add messages?

**Comment by michielbdejong** [comment]
@Vinnl that's due to https://github.com/solid/chat-pane/issues/28, you should use the small login button at the top

**Comment by Vinnl** [comment]
Cool, I figured that :)

I still couldn't see how to add messages, and after refreshing I get just this:

image

**Comment by kevin-howard-jd** [comment]
After a few page refreshes, I see the standard component for entering chat messages. 
Screen Shot 2020-01-06 at 8 12 51 AM

What is not obvious is how to create a new chat.

**Comment by michielbdejong** [comment]
It seems to work in Chrome but not in FF?

**Comment by michielbdejong** [comment]
Ah correction, it's not because of FF vs Chrome, it's because of https://github.com/solid/chat-pane/issues/31

**Comment by michielbdejong** [comment]
https://khoward.inrupt.net/settings/prefs.ttl and https://michielbdejong.inrupt.net/settings/prefs.ttl are public, but https://be-my-friend.inrupt.net/settings/prefs.ttl and https://vincent-test.inrupt.net/settings/prefs.ttl are not.

File: src/chat/infinite.js [hunk 1]
```diff
@@ -64,6 +64,29 @@ async function createIfNotExists (doc, contentType = 'text/turtle', data = '') {
   return response
 }
 
+function desktopNotification (str) {
+  // Let's check if the browser supports notifications
+  if (!('Notification' in window)) {
+    alert('This browser does not support desktop notification')
```
Comment (megoth) [inline src/chat/infinite.js]: An alert is perhaps a bit too intrusive? I know this probably won't happen to most users, but perhaps you should tone it down to a `console.warn` until we have something better in place?

Comment (megoth) [reply src/chat/infinite.js]: When you're good with this, you can merge it as you please.

Comment (michielbdejong) [reply src/chat/infinite.js]: done

**Comment by michielbdejong** [comment]
Restoring branch to make https://travis-ci.org/solid/solid-ui/jobs/634157446 pass

**Pull request merged**